core/clusters/clusterconfigs.go

```diff
@@ -128,6 +128,9 @@ func process(config *ClusterConfig, name, value, configmapname string) error {
 	case "exposeui":
 		config.ExposeWebUI = value
 		_, err = strconv.ParseBool(config.ExposeWebUI)
+		if err != nil {
```

Comment:
nit, this early return isn't needed here, they're included only in places where there might be additional processing after an error